packages/jest-cli/src/cli/args.js

```diff
@@ -102,11 +102,16 @@ export const options = {
       ' dependency information.',
     type: 'string',
   },
+  changedFilesSinceCommit: {
+    description:
+      'When used together with `--onlyChanged`, it runs tests ' +
+      'related to the current changes and the changes made in the last commit.',
+    type: 'string',
+  },
   changedFilesWithAncestor: {
     description:
       'When used together with `--onlyChanged`, it runs tests ' +
-      'related to the current changes and the changes made in the last commit. ' +
-      '(NOTE: this only works for hg repos)',
```

Comment:
Can this fix (thanks :D) be in a separate PR?

Reply:
Sure. I'll have a go at extracting it tomorrow (it's getting late here though).

Reply:
... okay so it turned out to be easier than I thought (https://github.com/facebook/jest/pull/5189) but I am genuinely off to bed now.